codegens/http/lib/code-http-converter.js

```diff
@@ -9,17 +9,17 @@ let utils = require('./util');
  */
 function getOptions () {
   return [{
-    name: 'Body trim',
+    name: 'Trim request body fields',
     id: 'trimRequestBody',
     type: 'boolean',
     default: true,
-    description: 'Trim request body fields'
+    description: 'Remove white space and additional lines that may affect the server\'s response'
```

Comment:
Shall we have the same description for http too ? Stating 'that may affect the server\'s response'  doesn't really apply for http messages @abhijitkane